src/main/java/sx/blah/discord/api/internal/DiscordClientImpl.java

```diff
@@ -456,6 +456,18 @@ public IUser getUserByID(String userID) {
 	}
 
 	@Override
+	public List<IUser> getUsersByName(String name) {
+		return getUsersByName(name, true);
```

Comment:
It should *not* default to true as all other getXsByName are case-sensitive only, including IGuild#getUsersByName